package.json

```
@@ -36,18 +37,25 @@
   },
   "devDependencies": {
     "@babel/core": "7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/plugin-proposal-object-rest-spread": "7.6.2",
     "@babel/plugin-transform-runtime": "7.6.2",
     "@babel/preset-env": "7.6.3",
     "@babel/preset-react": "7.6.3",
+    "@babel/preset-typescript": "^7.3.3",
     "@pika/pack": "0.5.0",
     "@pika/plugin-build-node": "0.7.1",
     "@pika/plugin-build-types": "0.7.1",
+    "@pika/plugin-build-umd": "^0.7.1",
     "@pika/plugin-build-web": "0.7.1",
+    "@pika/plugin-bundle-types": "0.7.1",
     "@pika/plugin-standard-pkg": "0.7.1",
+    "@pika/plugin-ts-standard-pkg": "0.7.1",
     "@storybook/react": "5.2.5",
     "@testing-library/jest-dom": "4.2.0",
     "@testing-library/react": "9.3.0",
+    "@typescript-eslint/eslint-plugin": "^2.1.0",
+    "@typescript-eslint/parser": "^2.1.0",
```

```suggestion
    "@typescript-eslint/parser": "2.1.0",
```

I added a `.npmrc` to hopefully avoid this in the future.